### PR TITLE
Added AAS into Jakarta EE 9 Web compatible list

### LINKED ISSUE #1120: Please add Apusic Application Server as Jakarta EE 9 Web Profile compatible
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://www.eclipse.org/membership/showMember.php?member_id=1397

**A product name and version**
AAS V10-EE9

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**
Web Profile

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
https://github.com/apusic/aas-tck/blob/master/aas.jpg

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
Kingdee Apusic cloud computing Co., Ltd.

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
https://github.com/eclipse-ee4j/jakartaee-platform/issues/328

**The URL where Compatible Software Product is available for download**
Currently we will not likely provide public downloading this version, and we will provide it to customer or specific user if they apply for.

### PR DESCRIPTION
**Pull request by linkfang**

### CODE REVIEW

File: data/compatible_products.yml [hunk 1]
```diff
@@ -37,6 +37,15 @@ sets:
 
     - title: Jakarta EE 9 Web Profile
       items:
+      - name: AAS
```
Comment (autumnfound) [inline data/compatible_products.yml]: This name should match the other Apusic product 'Apusic AAS'. There is some deduplication that happens, iirc

Comment (linkfang) [reply data/compatible_products.yml]: @autumnfound , OK, make sense. Thanks Martin!

**Review by chrisguindon — APPROVED** [review]
LGTM - Please make sure we have a +1 from Tanya before we push this live!

**Comment by TanjaObradovic** [comment]
+1

**Review by TanjaObradovic — APPROVED** [review]

**Pull request merged**